openpype/hosts/houdini/plugins/publish/validate_bgeo_file_sop_path.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+"""Validator plugin for SOP Path in bgeo isntance."""
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+
+
+class ValidateNoSOPPath(pyblish.api.InstancePlugin):
+    """Validate if SOP Path in BGEO instance exists."""
+
+    order = pyblish.api.ValidatorOrder
+    families = ["bgeo"]
+    label = "Validate BGEO SOP Path"
+
+    def process(self, instance):
+
+        import hou
+
+        node = hou.node(instance.data.get("instance_node"))
+        sop_path = node.evalParm("soppath")
+        if not sop_path:
+            raise PublishValidationError("Empty SOP Path found in "
+                                         "the bgeo isntance Geometry")
```

Comment:
```suggestion
            raise PublishValidationError("Empty SOP Path found in "
                                         "the bgeo instance Geometry")
```

Admittedly - the error message does look a bit vague to me. It should actually explain now where is the issue exactly and what could I do about it. We need to guide the artist in the right direction.

@antirotor could you improve both the validation messages?

Reply:
Agreed that this doesn't help the artist at all.